### PR TITLE
Fixed build status retrieving method from OpenShift

### PR DESCRIPTION
**Pull request by bamachrn**
* Fixed build status retrieval method due to upgardation of OpenShift

### CODE REVIEW

File: beanstalk_worker/worker_start_build.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 ch.setFormatter(formatter)
 logger.addHandler(ch)
 
-config_path = "/".join(sys.argv[0].split("/")[:-1])
+config_path = os.path.dirname(os.path.realpath(__file__));
```
Comment (navidshaikh) [inline beanstalk_worker/worker_start_build.py]: semicolon at the end of line?

File: beanstalk_worker/worker_start_build.py [hunk 2]
```diff
@@ -73,7 +73,7 @@ def start_build(job_details):
 
         debug_log(" change project to the desired one")
         command_change_project = "oc project " + namespace + \
-            " --config= " + config_path + " /node.kubeconfig"
+            " --config=" + config_path + "/node.kubeconfig"
```
Comment (navidshaikh) [inline beanstalk_worker/worker_start_build.py]: use `os.path.join` for joining the paths and add as variable.

File: beanstalk_worker/worker_start_build.py [hunk 3]
```diff
@@ -83,7 +83,7 @@ def start_build(job_details):
         out = run_command(command_start_build)
         debug_log(out)
 
-        build_details = out[0].rstrip()
+        build_details = out[0].split("\"")[1].rstrip()
```
Comment (navidshaikh) [inline beanstalk_worker/worker_start_build.py]: additional `"` ?

File: beanstalk_worker/worker_start_delivery.py [hunk 4]
```diff
@@ -82,7 +82,7 @@ def start_delivery(job_details):
         out = run_command(command_start_build)
         debug_log(out)
 
-        build_details = out[0].rstrip()
+        build_details = out[0].split("\"")[1].rstrip()
```
Comment (navidshaikh) [inline beanstalk_worker/worker_start_delivery.py]: is that an additional `"` ?

File: beanstalk_worker/worker_start_build.py (hunk 3 above)
Comment (navidshaikh) [inline beanstalk_worker/worker_start_build.py]: if that's the case then `split('"')` should do the needful

**Comment by dharmit** [comment]
LGTM, merging.

**Pull request merged**